.github/workflows/coverage.yml

```diff
@@ -0,0 +1,35 @@
+name:                           Coverage
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
In #94, I'm asking if it's okay to change the `on.push.branches` value from `[ master ]` to `'*'`.  The reasoning is that the ability to push to branches in the upstream repo is controlled via permissions & PR settings, but allowing builds on non-master branches allows CI to run on incremental pushes to feature branches in people's personal fork repos.  No extra cost b/c Github Actions free for open source, and I presume the process is lightweight FWIW.

Same question would apply here.

Reply:
FWIW, you get the same effect if you allow builds on `pull_request` for branch `master`. 

Pros (IMHO):
- Motivates people to create pull requests early, promoting work coordination.
- Allows a contributor to control whether to run the step or not, which is kinda nice in case of a string of commits.

Since pull requests can be set to "draft" mode, then they can be kept out of mainline list of ready-to-review PRs, so seems to have no downside.

The downside of an early PR is that others get to see your dirty work in progress, but I don't care much about that.

YMMV.

Reply:
Good point about encouraging more draft PRs, and about resource constraints.

Reply:
I'd like to separate the conversation about `branches` then.